packages/cli/lib/lib/webpack/sw_debug.js

```diff
@@ -0,0 +1,16 @@
+self.addEventListener('fetch', function(event){
+    const isJsonRequest = event.request.headers.get('content-type') === 'application/json';
+    const isPostRequest = event.request.method === 'POST';
+    event.respondWith(fetch(event.request).then(function(response){
+        return response;
+    }).catch(function(err){
+        console.error(err);
+        if (!navigator.onLine && isJsonRequest) {
```

Comment:
`navigator.onLine` check should not be here, it is highly unlikely that a dev machine will go offline